rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskQueryFilterParameter.java

```diff
@@ -2183,5 +2194,24 @@ private void validateFilterParameters() throws InvalidArgumentException {
       throw new InvalidArgumentException(
           "provided value of the property 'without-attachment' must be 'true'");
     }
+    if (ownerNull != null && (ownerNull.length() > 0)) {
```

Comment:
Do you still need the other check "checkExactParam". Or would this check be enough?